qa/workunits/rbd/rbd-ggate.sh

```diff
@@ -149,17 +149,20 @@ used=`rbd -p ${POOL} --format xml du ${IMAGE} |
 [ "${used}" -lt "${provisioned}" ]
 
 # resize test
+# resizing the underlying image for a GEOM ggate will stop the
+# ggate process servicing the device. So we can resize and test 
+# the disapearence of the device
 devname=$(basename ${DEV})
 size=$(geom gate list ${devname} | awk '$1 ~ /Mediasize:/ {print $2}')
 test -n "${size}"
-rbd resize ${POOL}/${IMAGE} --size $((SIZE * 2))M
+_sudo rbd resize ${POOL}/${IMAGE} --size $((SIZE * 2))M
```

Comment:
why does `rbd resize` need sudo?

Reply:
@trociny 
It will only run as either root or ceph if I test it
Now that could be due to access to ceph.conf, which I'll check.

Reply:
@trociny 
It was access to the keyring that was access restricted.